bitnami/oauth2-proxy/values.yaml

```diff
@@ -0,0 +1,473 @@
+  ## You can:
+  ##   - Relay on cert-manager to create it by setting `ingress.certManager=true`
+  ##   - Relay on Helm to create self-signed certificates by setting `ingress.tls=true` and `ingress.certManager=false`
+  ##
+  tls: false
+  ## @param ingress.extraHosts An array with additional hostname(s) to be covered with the ingress record
+  ## e.g:
+  ## extraHosts:
+  ##   - name: oaut2-proxy.local
+  ##     path: /
+  ##
+  extraHosts: []
+  ## @param ingress.extraPaths An array with additional arbitrary paths that may need to be added to the ingress under the main host
+  ## e.g:
+  ## extraPaths:
+  ## - path: /*
+  ##   backend:
+  ##     serviceName: ssl-redirect
+  ##     servicePort: use-annotation
+  ##
+  extraPaths: []
+  ## @param ingress.extraTls TLS configuration for additional hostname(s) to be covered with this ingress record
+  ## ref: https://kubernetes.io/docs/concepts/services-networking/ingress/#tls
+  ## e.g:
+  ## extraTls:
+  ## - hosts:
+  ##     - oaut2-proxy.local
+  ##   secretName: oaut2-proxy.local-tls
+  ##
+  extraTls: []
+
+## @section OAuth2 Proxy Image parameters
+
+## Bitnami OAuth2 Proxy image
+## ref: https://hub.docker.com/r/bitnami/redis/tags/
+## @param image.registry OAuth2 Proxy image registry
+## @param image.repository OAuth2 Proxy image repository
+## @param image.tag OAuth2 Proxy image tag (immutable tags are recommended)
+## @param image.pullPolicy OAuth2 Proxy image pull policy
+## @param image.pullSecrets OAuth2 Proxy image pull secrets
+## @param image.debug Enable image debug mode
+##
+image:
+  registry: docker.io
+  repository: bitnami/oauth2-proxy
+  tag: 7.1.3-debian-10-r25
+  ## Specify a imagePullPolicy
+  ## Defaults to 'Always' if image tag is 'latest', else set to 'IfNotPresent'
+  ## ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images
+  ##
+  pullPolicy: IfNotPresent
+  ## Optionally specify an array of imagePullSecrets
+  ## Secrets must be manually created in the namespace
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+  ## e.g:
+  ## pullSecrets:
+  ##   - myRegistryKeySecretName
+  ##
+  pullSecrets: []
+  ## Enable debug mode
+  ##
+  debug: false
+
+## @section Oauth client configuration specifics
+
+## Configuration section
+## @param config.clientID OAuth client ID
+## @param config.clientSecret OAuth client secret
+## @param config.cookieSecret OAuth cookie secret
+## @param config.cookieName OAuth cookie name
+## @param config.google Google service account
+## @param config.configFile Default configuration
+##
+config:
+  clientID: "XXXXXXX"
+  clientSecret: "XXXXXXXX"
+  ## Create a new secret with the following command openssl rand -base64 32 | head -c 32 | base64
+  ## Use an existing secret for OAuth2 credentials (see secret.yaml for required fields)
+  ## e.g:
+  ## existingSecret: secret
```

Comment:
We don't leave commented out values anymore. Make sure you add it with an empty value. You may leave the example